README.md

```diff
@@ -190,6 +192,32 @@ OpenID Connect requires setting up the  `signing_key`. The signing key can be
 generated via a command like
 `openssl genpkey -algorithm RSA -out key.pem -pkeyopt rsa_keygen_bits:2048`. 
 
+#### OpenID Connect with Rocket.Chat
+
+To configure Rocket.Chat to use OpenID Connect, you must setup a custom Oauth provider.
+
+In the admin interface, you should scroll down to Oauth and open that. At the top, you should click "Add custom oauth". At this page, you will need to configure a few things:
+
+- URL: The URL to your EyeDP installation
+- Token Path: `/oauth/token`
+- Token Sent Via: `Payload`
+- Identity Token Sent Via: `Header`
+- Identity Path: `/oauth/userinfo`
+- Authorize Path: `/oauth/authorize`
+- Scope: `openid`
+- Param Name for access token: `code`
+- Id: The UID configured in EyeDP
+- Secret: Secret configured in EyeDP
+- Login Style: Redirect
+- Username field: `username`
+- Email field: `email`
+- Name field: `name`
+- Roles/Groups field name: `groups`
+
+It is also possible to enable Merging Roles from SSO, Merging users, and Showing Button on Login Page.
+
```

Comment:
this text could benefit from a screenshot

Reply:
Added one here as well as for the SAML example below